autoload/nvimterm.vim

```diff
@@ -47,6 +50,40 @@ function! nvimterm#delete_buffers(all) "{{{1
 	let multi = 'term deleted'
 	echomsg delete_count delete_count <= 1 ? single : multi
 endfunction
+
+function! s:closeWinTerm() "{{{1
+	for n in range(1, winnr('$'))
+		let buffer_type = getwinvar(n, '&buftype')
```

Comment:
**Fix future:** If it runs on a terminal that is not a toggle terminal, it closed.